backend/fms_core/viewsets/_constants.py

```diff
@@ -242,6 +229,24 @@
     "project_name": CATEGORICAL_FILTERS_LOOSE,
     "metric_report_url": CATEGORICAL_FILTERS_LOOSE,
     "experiment_run": FK_FILTERS,
+    "readsets__validation_status": CATEGORICAL_FILTERS_LOOSE,
+    "readsets__release_status": CATEGORICAL_FILTERS_LOOSE,
+
+}
+
+_experiment_run_filterset_fields: FiltersetFields = {
+    "id": PK_FILTERS,
+    "name": CATEGORICAL_FILTERS_LOOSE,
+    "start_date": DATE_FILTERS,
+    "run_type": FK_FILTERS,
+    "instrument": FK_FILTERS,
+    "container": FK_FILTERS,
+    "run_processing_launch_time": DATE_FILTERS,
+
+    **_prefix_keys("run_type__", _run_type_filterset_fields),
+    **_prefix_keys("instrument__", _instrument_filterset_fields),
+    **_prefix_keys("container__", _container_filterset_fields),
+    **_prefix_keys("datasets__", _dataset_filterset_fields),
```

Comment:
All these filterset might be a bit overkill for the need we have ... We basically want to know if the experiment_run is processed (at least one dataset validation at available), validated (No dataset validation available, all validation status either passed or failed), released (all released status at released or blocked). This much business logic does not sit well in a frontend ... we can make a backend filter for experiment_run progression state. The additional filters for release or blocked, passed or failed would be better elsewhere ... they would be part of another PR.